enterprise/backend/test/metabase_enterprise/serialization/upsert_test.clj

```diff
@@ -128,4 +128,4 @@
   (is (= true
          (#'upsert/has-post-insert? User)))
   (is (= false
-         (#'upsert/has-post-insert? Table))))
+         (#'upsert/has-post-insert? Metric))))
```

Comment:
How's this related?

Reply:
This was also in the feature branch already.

But essentially I added a post-insert function for Table so this test started failing. It's just testing the functionality of `has-post-insert?` so it doesn't matter what model it uses, as long as it doesn't have a post-insert function defined